src/azure-cli-core/setup.py

```diff
@@ -51,7 +51,7 @@
     'humanfriendly>=4.7,<10.0',
     'jmespath',
     'knack==0.8.0rc2',
-    'msal~=1.9',
+    'msal>=1.10.0,<2.0.0',
```

Comment:
Given the stability of MSAL, we can relax it to `<2.0.0`. This is also what Azure Identity requires:

https://github.com/Azure/azure-sdk-for-python/blob/db4892a29c45fd9d65da1319d65ffd3269f13898/sdk/identity/azure-identity/setup.py#L77

```py
"msal<2.0.0,>=1.7.0",
```